Source/UI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -38,17 +37,39 @@ final class BillingFormViewController: UIViewController {
     private var notificationCenter = NotificationCenter.default
 
     // MARK: - UI elements
+    private(set) lazy var cancelItem: UIBarButtonItem = {
+        let item = UIBarButtonItem(title: viewModel.style.header.cancelButton.text,
+                         style: .plain,
+                         target: self,
+                         action: #selector(cancelButtonIsPressed))
+        item.tintColor = viewModel.style.header.cancelButton.textColor
+        return item
+    }()
 
-    private lazy var headerView: UIView? = {
-        delegate?.getViewForHeader(sender: self)?.disabledAutoresizingIntoConstraints() ?? UIView()
+    private(set) lazy var doneItem: UIBarButtonItem = {
+        let item = UIBarButtonItem(title: viewModel.style.header.doneButton.text,
+                         style: .plain,
+                         target: self,
+                         action: #selector(doneButtonIsPressed))
+        item.isEnabled = viewModel.data != nil
+        item.tintColor = viewModel.data == nil ? viewModel.style.header.doneButton.disabledTextColor :
+        viewModel.style.header.doneButton.textColor
```

Comment:
This line should have some indentation, although Xcode doesn't automatically do it. As its aligned it looks like an individual line